pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -516,6 +520,31 @@ class ESCPOSRenderer(private val dialect: Dialect, private val receipt: JSONObje
     }
 
     private fun mode(font: String = "a", emph: Boolean = false, doubleheight: Boolean = false, doublewidth: Boolean = false, underline: Boolean = false) {
+        if (dialect == Dialect.StarPRNT) {
+            // font
+            out.add(ESC)
+            out.add(RS)
+            out.add('F'.toByte())
+            when(font) {
+                "a" -> out.add(0)
+                "b" -> out.add(1)
+                "c" -> out.add(2)
+            }
+            // emph
+            out.add(ESC)
+            out.add((if (emph) 'E' else 'F').toByte())
+            // double
+            out.add(ESC)
+            out.add('i'.toByte())
+            out.add(if (doubleheight) 1 else 0)
+            out.add(if (doublewidth) 1 else 0)
+            // underline
+            out.add(ESC)
+            out.add('-'.toByte())
+            out.add(if (underline) 1 else 0)
+            return
+        }
+
```

Comment:
Not a big change and probably by no means absolutely necessary: But perhaps instead of having an `if` section for Star that then returns with the normal code below, we can make this a `when()` with a `default`-branch?